{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -7,31 +7,43 @@
 from .__about__ import __version__
 
 
-################# Configuration
-config = {
-    # Add here your new settings
-    "defaults": {
-        "VERSION": __version__,
-    },
-    # Add here settings that don't have a reasonable default for all users. For
-    # instance: passwords, secret keys, etc.
-    "unique": {
-        # "SECRET_KEY": "{{ '{{' }} 24|random_string {{ '}}' }}",
-    },
-    # Danger zone! Add here values to override settings from Tutor core or other plugins.
-    "overrides": {
-        # "PLATFORM_NAME": "My platform",
-    },
-}
+################# CONFIGURATION
+hooks.Filters.CONFIG_DEFAULTS.add_items(
+    [
+        # Add here your new settings that have default values.
+        # Each new setting is a pair: (setting_name, default_value).
+        # Prefix your setting names with '{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_'.
+        ("{{ cookiecutter.plugin_name|upper|replace('-', '_') }}_VERSION", __version__),
+    ]
+)
```

Comment:
A newline between filters may make it easier to read/understand for developers.

Reply:
👍🏻 

I did this for all filters except IMAGE_{PUSH,PULL} because those are referenced by the same comment.